src/qt/peertablemodel.cpp

```diff
@@ -113,8 +113,10 @@ QVariant PeerTableModel::data(const QModelIndex &index, int role) const
         case NetNodeId:
             return (qint64)rec->nodeStats.nodeid;
         case Address:
-            // prepend to peer address down-arrow symbol for inbound connection and up-arrow for outbound connection
-            return QString(rec->nodeStats.fInbound ? "↓ " : "↑ ") + QString::fromStdString(rec->nodeStats.addrName);
+            return QString::fromStdString(rec->nodeStats.addrName);
+        case Direction:
+            //: Connection direction. Also used in ConnectionTypeToQString() and CONNECTION_TYPE_DOC.
+            return QString(rec->nodeStats.fInbound ? tr("Inbound") : tr("Outbound"));
```

Comment:
Can we use `IN` and `OUT` in caps instead of Inbound and Outbound?

Reply:
Here is how that would look, not a bad idea. But, a) I think this change is ok as is b) would like to hear other opinions.

![Screen Shot 2021-04-30 at 3 09 34 AM](https://user-images.githubusercontent.com/23396902/116660814-89938380-a961-11eb-9bd4-a37ee76ab85e.png)

Reply:
The goal is to unify the peers tab Direction and Type columns with the Direction/Type row in the peer details sidebar, so if the side bar displays "Outbound Full Relay", the main table also displays "Outbound Full Relay" with the ability to sort by either Direction or Type.

Reply:
I tried to communicate it in the PR description.

Reply:
(I also tried "In/Out" while working on this PR, and the horizontal space isn't reduced unless the column header is "Dir" instead of "Direction". But the other headers are full words.)